lsp-bridge.el

```diff
@@ -2531,6 +2541,27 @@ We need exclude `markdown-code-fontification:*' buffer in `lsp-bridge-monitor-be
                              (acm-get-input-prefix)
                              language))))
 
+(defun lsp-bridge-jupyter-complete ()
+  (interactive)
+  (unless (fboundp 'jupyter-org-completion-at-point)
+    (require 'jupyter)
+    (require 'jupyter-org-client)
+    (require 'ob-jupyter))
+  (setq-local acm-backend-jupyter-items nil)
+  (let* ((candidates (all-completions "" (nth 2 (jupyter-org-completion-at-point))))
+         (candidates-with-metadata (mapcar (lambda (candidate)
+                                             (let ((text (substring-no-properties candidate)))
+                                               (list :key text
+                                                     :icon ""
+                                                     :label text
+                                                     :displayLabel text
+                                                     :annotation "jupyter"
+                                                     :backend "jupyter")))
+                                           candidates)))
+    (when candidates
+      (lsp-bridge-search-backend--record-items "jupyter" candidates-with-metadata)
+      )))
```

Comment:
Those code should move to acm-backend-jupyter and rename with function name acm-backend-jupyter-record

Reply:
If I rename these codes to function acm-backend-jupyter-record in acm-backend-jupyter.el file, how can I call it in `lsp-bridge-complete-other-backends` function?
Sorry, I just do not know how to trigger the `acm-backend-jupyter-record` function. It seems it is called by `.py` files for other backend.

If I call it with the following code, the pop menu candidate will not update as I typing：

``` elisp
    (when (and acm-enable-jupyter
               (lsp-bridge-process-live-p)
               (and lsp-bridge-enable-org-babel
                    (eq major-mode 'org-mode)
                    (not (lsp-bridge-is-org-temp-buffer-p)))
               (org-in-src-block-p 'inside)
               (string-prefix-p "jupyter" (plist-get (car (cdr (org-element-context))) :language)))
      ;; (lsp-bridge-call-async "jupyter" current-word)
      (acm-backend-jupyter-record nil)
      )
```

```elisp
(defun acm-backend-jupyter-record (candidates)
  (unless (fboundp 'jupyter-org-completion-at-point)
    (require 'jupyter)
    (require 'jupyter-org-client)
    (require 'ob-jupyter))
  (setq-local acm-backend-jupyter-items nil)
  (let* ((candidates (all-completions "" (nth 2 (jupyter-org-completion-at-point))))
         (candidates-with-metadata (mapcar (lambda (candidate)
                                             (let ((text (substring-no-properties candidate)))
                                               (list :key text
                                                     :icon ""
                                                     :label text
                                                     :displayLabel text
                                                     :annotation "jupyter"
                                                     :backend "jupyter")))
                                           candidates)))
    (when candidates
      (lsp-bridge-search-backend--record-items "jupyter" candidates-with-metadata))))
```